index.php

```diff
@@ -7,6 +7,42 @@
 // Creates a variable which points to the correct database and gives username and password
 $db = new PDO('mysql:host=db;dbname=dinosaurs;', 'root', 'password');
 
+/**
+ * Checks if the form has been submitted and if so returns the dinos from the submitted search. If not, returns all dinos
+ *
+ * @param PDO $db database
+ * @return string of dinos to be displayed
+ */
+function checkIfSearched(PDO $db): string {
+    if (isset($_GET['submit'])) {
+        if (isset($_GET['search'], $_GET['filter'])) {
+            return Museum::displayAllDinos(DinosaurHydrator::getSearchedDinos($db, $_GET['search'], $_GET['filter']));
+        } else {
+            return Museum::displayAllDinos(DinosaurHydrator::getAllDinos($db));
+        }
+    } else {
+        return Museum::displayAllDinos(DinosaurHydrator::getAllDinos($db));
+    }
+}
+
+/**
+ * Checks if the form has been submitted and if so returns the value the user searched for, otherwise returns search
+ *
+ * @return string
+ */
+function searchedValue(): string {
+    if (isset($_GET['submit'])) {
```

Comment:
again an if with && logic and no else's would be dryer

Reply:
Awesome :)